lib/SIL/Verifier/LinearLifetimeCheckerPrivate.h

```diff
@@ -209,6 +209,7 @@ class LLVM_LIBRARY_VISIBILITY LinearLifetimeChecker::ErrorBuilder {
 
     llvm::errs() << "Found ownership error?!\n";
     assert(0 && "triggering standard assertion failure routine");
+    return true;
```

Comment:
`llvm_unreachable("message")` is the canonical way to write `assert(0 && "message");`, and should avoid this failure

Reply:
Will `llvm_unreachable()` avoid the PrettyStackTrack swallowing?
https://github.com/apple/swift/pull/32500

Reply:
Yea. It will be fine. I don't remember why I did it this way. I was pretty working quickly.

Reply:
The problem in that other one was llvm::report_fatal_error.